make-distribution.sh

```diff
@@ -147,7 +147,7 @@ if [[ ! "$JAVA_VERSION" =~ "1.6" && -z "$SKIP_JAVA_TEST" ]]; then
   echo "Output from 'java -version' was:"
   echo "$JAVA_VERSION"
   read -p "Would you like to continue anyways? [y,n]: " -r
-  if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+  if [[ ! "$REPLY" =~ ^[Yy]$ ]]; then
```

Comment:
Since we are using the Bash-specific `[[ ... ]]` syntax here, there is actually no word splitting, so the original line should be safe. But I don't think it hurts to quote `$REPLY` anyway.